### PR TITLE
Include system contracts

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: system_contracts.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+- **variable length**: *eWASM contract code*
+
+Output:
+- **variable length**: *eWASM contract code*
+
+## EVM Transcompiler
+
+Address: `0x000000000000000000000000000000000000000b`
+
+This is optional. It is provided for pure eWASM VMs to support EVM1 contracts.
```
Comment (wanderer) [inline system_contracts.md]: optional is a bit misleading here

Comment (axic) [inline system_contracts.md]: Why? They are optional as in not a mandatory part of a client if it opts to implement EVM1 natively.

Comment (wanderer) [inline system_contracts.md]: yes but they will still need to implement the logic to run the EVM1 when a call goes to that address.

Comment (axic) [inline system_contracts.md]: I don't think this is related to the precompile and the transcompiler.

**Pull request merged**